tests/profiles/universal_test.py

```diff
@@ -509,39 +549,46 @@ def test_check_required_tables():
     # TODO: Maybe we could someday emit a more explicit
     #       message to the users regarding that...
     ttFont.reader.tables["fvar"] = "foo"
-    assert_results_contain(check(ttFont),
-                           FAIL, "required-tables",
-                           'with fvar but no STAT...')
+    msg = assert_results_contain(check(ttFont),
+                                 FAIL, "required-tables",
+                                 'with fvar but no STAT...')
+    assert "STAT" in msg
 
     del ttFont.reader.tables["fvar"]
     ttFont.reader.tables["STAT"] = "foo"
-    assert_PASS(check(ttFont),
-                'with STAT on a non-variable font...')
+    msg = assert_PASS(check(ttFont),
+                      'with STAT on a non-variable font...')
+    assert msg == "Font contains all required tables."
 
     # and finally remove what we've just added:
     del ttFont.reader.tables["STAT"]
+
     # Now we remove required tables one-by-one to validate the FAIL code-path:
-    for required in required_tables:
+    for required in REQUIRED_TABLES + ["glyf"]:
```

Comment:
Please add a comment to the code-test explaining the `+ ["glyf"]` here.

I'm being picky here because Font Bakery's mission is to educate its users, which also includes anyone who reads its source code. The rationale strings and well documented code-tests are part of this mission.

Reply:
Comments added in 3e08b6911ab98d158b2dca3a09e2b5eb5a62b106